web/src/beta/hooks/navigationHooks.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from "react";
+import { useNavigate } from "react-router-dom";
+
+export type Tab = "scene" | "story" | "widgets" | "publish";
+
+export const useEditorNavigation = ({ sceneId }: { sceneId?: string }) => {
+  const navigate = useNavigate();
+
+  return useCallback(
+    (tab: Tab) => {
+      navigate(tab !== "scene" ? `/scene/${sceneId}/${tab}` : "");
+    },
+    [sceneId, navigate],
+  );
+};
+
+export const useNavigationHooks = () => {
+  return {
+    useEditorNavigation,
+  };
+};
```

Comment:
BTW I'm concerned that bundling like this will increase bundle size.
It might split each hooks.

Reply:
So you think maybe its better/okay to just export everything individually, right? I think that is reasonable and maybe sticks to the initial idea better.

Reply:
Yes, I think it's better to export individual function.

Reply:
Done in [Update hooks](https://github.com/reearth/reearth/pull/444/commits/0fb5084f288137bf2953f2e99e7e2d6908b3a460) commit